pyproject.toml

```diff
@@ -165,3 +165,29 @@ image_cache_dir = "tests/plotting/image_cache"
 
 [tool.mypy]
 ignore_missing_imports = true
+
+[tool.ruff]
+exclude = [
```

Comment:
I would suggest also to exclude a possible `venv` and `.venv` directories, @tkoyama010 @banesullivan.